scanner/templates/report.jinja2

```diff
@@ -26,57 +28,65 @@
                 <h2><a href="/">Heimdall Report</a></h2>
             </div>
             <div class="six columns">
-                <p class="text-right"><small>powered by mythril</small>
+                <p class="text-right">
+                    <small>powered by mythril</small>
             </div>
         </div>
-        
-        <h4><a href="https://github.com/{{ owner }}/{{ repo }}" target="_blank"><i class="fab fa-github"></i> {{ owner }}/{{ repo }}</a></h4>
+
+        <h4><a href="https://github.com/{{ owner }}/{{ repo }}" target="_blank"><i
+                class="fab fa-github"></i> {{ owner }}/{{ repo }}</a></h4>
         <hr>
-        {% if mythril.report %}
-            {% for key, item in mythril.report.items() %}
-                <h3>{{ item.file }} {% if not item.data.issues %}<i class="fa fa-check"></i>{% endif %}</h3>
-                {% for issue in item.data.issues %}
-                    <div class="{% if issue.type == 'Warning' %}bg-warning{% endif %}">
-                        <div class="row">
-                            <div class="eight columns">
-                                <h5 class="card-title">Contract {{ issue.contract }} Issue: {{ issue.title }}</h4>
-                            </div>
-                            <div class="four columns">
-                                <p class="text-right">{{ issue.type }}</p>
+        {% if message == None %}
```

Comment:
maybe `if not message` ? But I think that variable should be `error` -> `if not error`